pkg/cosign/tuf/client.go

```diff
@@ -44,8 +43,7 @@ const (
 	DefaultRemoteRoot = "sigstore-tuf-root"
 )
 
-//go:embed repository/*.json
-//go:embed repository/targets/*.pem repository/targets/*.pub
```

Comment:
What happened to these?

Reply:
embedding a directory copies everything recursively, except for files beginning with `.` or `_`. We ended up missing a bunch of embedded files due to a misunderstanding about the behavior of `*` in the directive